### ISSUE TITLE
How to get the cluster label for new data X

### ISSUE THREAD

**Issue by hana9090** [fferroni/DEC-Keras#12]
After the cluster is formed, how to know which cluster does the new data belong to?
And also how to retrieve all images which rely on the same cluster?

**Comment by anuraagvaidya** [comment]
I'd like to know this as well.

**Comment by digivi13** [comment]
to find out which cluster new data belongs to, get the embedded representations for all the images and perform a PCA on the entire set of embedded training data. Then use model.encoder.predict(X) or model.get_layer(index=0).predict(X) (depending on if you are loading a saved model) to get the embedded representation of the new image. Use the same PCA object (important!) on the embedding of the new image to get the coordinates and plot it along with the training data (different colors of course)
